config/samples/_v1alpha1_turbine.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Turbine
+metadata:
+  name: turbine-ping-pong
+spec:
+  sidecar:
+    name: turbine
+    image: hazelcast/turbine-sidecar
+  hazelcast:
+    cluster:
+      name: hazelcast
+      namespace: hazelcast
```

```suggestion
      namespace: default
```
or namespace should not be configured here then kubebuilder default value will be used.